src/Scene.scss

```diff
@@ -0,0 +1,14 @@
+@import 'variables';
+
+.Scene__drawing-area {
+    display: inline-block;
+    box-sizing: border-box;
+    width: 100%;
+    height: 55.5%;
```

Comment:
We can use `height: auto` here to maintain the aspect ratio of the SVG, without having to specify it explicitly in the CSS.